### DISCUSSION TITLE
Unable to start (mine/idle) after CloseLoots

### DISCUSSION THREAD

**Discussion by liamking9** [coccoinomane/crabada.py#122] [Q&A]
Hi Everyone,

i was trying out the semi-auto looting bot and like what i see so far.

however, i noticed that after 1 or 2 loots being closed automatically for a Team, the said team is no longer able to loot/mine.

I can confirm that :
1. My team is idle (appears free/idle in Manage Teams section)
2. There is sufficient loot point to begin next loot mission
3. Unable to withdraw crabs from the affected team (Tx unable to pass through)

Anyone has faced the same issues before? i was thinking could there be an issue with how the loot mission is being "settled" ?

My current solution/woprkaround to this is to re-run the auto mining to get the affected team (which appears idle in manage Teams) to mine, which seems to resolve the problem of unable to take any action.

Regards,
Liam

**Comment by coccoinomane** [comment]
Hello @liamking9 and sorry for the late reply!

Did you eventually solve your issue? Please consider that Crabada.py bot currently does not support sending teams looting.

Cheers,
Cocco
